flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/FlinkKinesisProducer.java

```diff
@@ -57,6 +55,9 @@
 	/** Properties to parametrize settings such as AWS service region, access key etc. */
 	private final Properties configProps;
 
+	/** Configuration for KinesisProducer. */
+	private final KinesisProducerConfiguration producerConfig;
```

Comment:
`KinesisProducerConfiguration` isn't serializable. We need to make it `transient`.